autoflip_calculators/text_detection_calculator.cc

```diff
@@ -59,28 +72,32 @@ class TextDetectionCalculator : public CalculatorBase {
   ::mediapipe::Status Process(mediapipe::CalculatorContext* cc) override;
 
  private:
-  double NormalizeX(const int pixel);
-  double NormalizeY(const int pixel);
   // Decode the outputs of the EAST neural network
   void DecodeBoundingBoxes(const cv::Mat& socres, const cv::Mat& geometry, float scoreTresh,
-                         std::vector<RotatedRect>& detections, std::vector<float>& confidences);
+                         std::vector<cv::RotatedRect>& detections, std::vector<float>& confidences);
+
+  // Detect the text.
+  void DetectText(const cv::Mat& frame, cv::dnn::Net detector,
+                cv::Mat& scores, cv::Mat& geometry);
```

Comment:
When you are passing by reference, use const.  So:
const cv::Mat& scores, const cv::Mat& geometry